src/app/components/Image/index.stories.tsx

```diff
@@ -18,7 +18,7 @@ export const ResponsiveImage = () => (
     <Image
       alt="A penguin stands on an ice floe"
       src="https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg"
-      srcSet="https://ichef.bbci.co.uk/news/240/cpsprodpb/164AF/production/_110911319_antartica.jpg 240w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
+      srcSet="https://ichef.bbci.co.uk/news/240/cpsprodpb/164AF/production/_110911319_antartica.jpg 660w, https://ichef.bbci.co.uk/news/624/cpsprodpb/164AF/production/_110911319_antartica.jpg 624w"
```

Comment:
this doesn't look right to me.

Isn't this saying when the image is 660px wide, we serve up an image 240px wide?

Reply:
tbh, the image srcset and sizes isn't my strongpoint, so I may be way off the mark

Reply:
Just forgot to change this from some visual tests we were doing in draft.